### DISCUSSION TITLE
Airflow Audit Logs | Privilege Logs | Admin activities logged in or not ?

### DISCUSSION THREAD

**Discussion by github-aman** [apache/airflow#23581] [Q&A]
Dear All,

I want to ask regarding the airflow privilege logs whether such logs/events stores at airflow side or not or any command to check or possible through UI.

Airflow Version:- 2.1.0(AWS ECS Fargate Setup)

Below are the items I am looking for:-

1. Event related to any new group creation/deletion at airflow side from UI.
2. Event related to new/custom group assigning to the airflow user.
3. Event related to the existing group removal of any user & adding again new/different one.
4. Event related to the user_id(majorly admin) who is performing above tasks.
5. logs/event related to the user authentication from UI where we can verify the [user_id, ip of machine, location, timestamp, etc.]

Let me know if someone can help me on above or how to validate above logs/events if it is already there & i am missing with something !!

Thanks,
Aman Thaledi

**Comment by potiuk** [comment]
How about this one: https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html?highlight=audit#audit-view-excluded-events

**Reply by github-aman** [reply]
@potiuk Thanks for the quick response !!

I have gone through the shared page, but i believe its majorly related to the dag events rather than the user activity.
Mostly I am looking for the events related to user activities like user creation, new role creation, deletion of any group, any permission change.

Let me know if something can be there & can be captured as well !!

Thanks,
Aman Thaledi

**Reply by potiuk** [reply]
I think adding PR about it might be a good idea of it's missing
